sbpy/activity/gas/tests/test_core.py

```diff
@@ -71,56 +74,24 @@ def test_fluorescence_band_strength_error():
         fluorescence_band_strength("OH 0-0", source="asdf")
 
 
-def test_gascoma_scipy_error(monkeypatch):
-    monkeypatch.setattr(core, "scipy", None)
-    test = Haser(1 / u.s, 1 * u.km / u.s, 1e6 * u.km)
-    with pytest.raises(sbe.RequiredPackageUnavailable):
-        test._integrate_volume_density(1e5)
-
-    with pytest.raises(sbe.RequiredPackageUnavailable):
-        aper = core.CircularAperture(1000 * u.km)
-        test._integrate_column_density(aper)
-
-
 class TestHaser:
-    def test_volume_density(self):
-        """Test a set of dummy values."""
-        Q = 1e28 / u.s
-        v = 1 * u.km / u.s
-        parent = 1e4 * u.km
-        daughter = 1e5 * u.km
-        r = np.logspace(1, 7) * u.km
-        n = Haser(Q, v, parent, daughter).volume_density(r)
-        rel = (
-            daughter
-            / (parent - daughter)
-            * (np.exp(-r / parent) - np.exp(-r / daughter))
-        )
-        # test radial profile
-        assert np.allclose((n / n[0]).value, (rel / rel[0] * (r[0] / r) ** 2).value)
-
-        # test parent-only coma near nucleus against that expected for
-        # a long-lived species; will be close, but not exact
-        n = Haser(Q, v, parent).volume_density(10 * u.km)
-        assert np.isclose(
-            n.decompose().value,
-            (Q / v / 4 / np.pi / (10 * u.km) ** 2).decompose().value,
-            rtol=0.001,
-        )
-
     def test_column_density_small_aperture(self):
         """Test column density for aperture << lengthscale.
 
         Should be within 1% of ideal value.
 
         """
+
+        pytest.importorskip("scipy")
```

Comment:
Just to confirm: Is this and other `importorskip` to test various package set up?

Reply:
Yes, if scipy is not available, the test is skipped.